libs/BehaviorKit/include/BehaviorKit.h

```diff
@@ -41,7 +41,7 @@ class BehaviorKit
 	void sprinkles();
 	void rainbow();
 
-	void bleConnection();
+	void bleConnection(bool isCharging);
```

Comment:
je suis pas très fan de l'idée que le bleConnection sait si on est charging ou pas. je préfère

```suggestion
void bleConnection(bool with_video);
```